### PR TITLE
feat(StatusTagSelector): Adding component documentation

### PR DESCRIPTION
**Pull request by alexandraB99**
Part of #616

### Checklist

- [x] follow [conventional commits](https://www.conventionalcommits.org/en/v1.0.0/)
  - the scope should be the component's name e.g: `feat(StatusListItem): ... `
  - when adding new components, the scope is the module e.g: `feat(StatusQ.Controls): ...`
- [x] add documentation if necessary (new component, new feature)
- [ ] update sandbox app
  - in case of new component, add new component page
  - in case of new features, add variation to existing component page
  - nice to have: add it to the demo application as well
- [ ] test changes in both light and dark theme?
- [ ] is this a breaking change?
    - [ ] use the dedicated `BREAKING CHANGE` commit message section
    - [ ] resolve breaking changes in [status-desktop](https://github.com/status-im/status-desktop)
        - [ ] (pre-merge) adapt code to breaking changes
        - [ ] (post-merge) update StatusQ submodule pointer
- [ ] test changes in [status-desktop](https://github.com/status-im/status-desktop)

### CODE REVIEW

File: src/StatusQ/Components/StatusTagSelector.qml [hunk 1]
```diff
@@ -6,29 +6,119 @@ import QtGraphicalEffects 1.0
...
+    */
     property ListModel sortedList: ListModel { }
+    /*!
+        \qmlproperty ListModel StatusTagSelector::namesModel
+        This property holds the asorted names model.
+    */
     property ListModel namesModel: ListModel { }
 
+    /*!
+        This function is used to find an entry in a model.
```
Comment (noeliaSD) [inline src/StatusQ/Components/StatusTagSelector.qml:112]: You can use [\qmlmethod](https://doc.qt.io/qt-5/13-qdoc-commands-topics.html#qmlmethod-command) tag for functions!

**Review by noeliaSD — CHANGES_REQUESTED** [review]

File: src/StatusQ/Components/StatusTagSelector.qml [hunk 2]
```diff
@@ -6,29 +6,119 @@ import QtGraphicalEffects 1.0
...
 
+/*!
+     \qmltype StatusTagSelector
+     \inherits Item
+     \inqmlmodule StatusQ.Components
+     \since StatusQ.Components 0.1
+     \brief Displays a tag selector component together with a list of where to select and add the tags from.
+     Inherits \l{https://doc.qt.io/qt-5/qml-qtquick-item.html}{Item}.
+
+     The \c StatusTagSelector displays a a list of asorted elements together with a text input where tags are added. As the user
```
Comment (PascalPrecht) [inline src/StatusQ/Components/StatusTagSelector.qml:17]:
```suggestion
     The \c StatusTagSelector displays a list of asorted elements together with a text input where tags are added. As the user
```

**Review by PascalPrecht — COMMENTED** [review]
Just a tiny typo, otherwise good stuff!

File: src/StatusQ/Components/StatusTagSelector.qml (hunk 2 above)
Comment (alexandraB99) [reply src/StatusQ/Components/StatusTagSelector.qml]: done

File: src/StatusQ/Components/StatusTagSelector.qml (hunk 1 above)
Comment (alexandraB99) [reply src/StatusQ/Components/StatusTagSelector.qml:113]: done

**Review by noeliaSD — APPROVED** [review]

**Review by iurimatias — APPROVED** [review]

**Pull request merged**